### PR TITLE
UIINREACH-21: INN-Reach Record Contribution Settings: Configure FOLIO Libraries and Shelving Locations to INN-Reach Locations Mapping

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
- deleted CONTRIBUTION_CRITERIA_PATH
- renamed useServers hook
- moved sections to constants folder
- created a new setting "FOLIO to INN-Reach Locations" ([UIINREACH-21](https://issues.folio.org/browse/UIINREACH-21))
- covered with tests
- changed URL for POST request for ContributionCriteria component

### CODE REVIEW

File: src/constants/contributionCriteria.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
-import { SETTINGS_PATH } from './base';
```
Comment (Godlevskyi) [inline src/constants/contributionCriteria.js:1]: Please rename the file to contribution-criteria.js

File: src/constants/folioToInnReachLocations.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const FOLIO_TO_INN_REACH_LOCATIONS_ROUTE = 'folio-to-inn-reach-locations';
+
+export const FOLIO_TO_INN_REACH_LOCATIONS = {
```
Comment (Godlevskyi) [inline src/constants/folioToInnReachLocations.js:3]: Please rename the file to folio-to-innreach-locations.js
Also rename to FOLIO_TO_INN_REACH_LOCATIONS to FOLIO_TO_INN_REACH_LOCATION_FIELDS

File: src/constants/sections.js [hunk 3]
```diff
@@ -4,11 +4,13 @@ import {
...
   ContributionCriteriaRoute,
-} from '../../routes';
+  FolioToInnReachLocationsRoute,
+} from '../settings/routes';
 import {
   CONTRIBUTION_CRITERIA_ROUTE,
+  FOLIO_TO_INN_REACH_LOCATIONS_ROUTE,
   RECORD_CONTRIBUTION,
-} from '../../../constants';
+} from './index';
```
Comment (Godlevskyi) [inline src/constants/sections.js:13]: Please do not import from index file directly
You can import like this'../constants' or to import separately

File: src/constants/contributionCriteria.js (hunk 1 above)
Comment (Dmytro-Melnyshyn) [reply src/constants/contributionCriteria.js:1]: Thanks, fixed

File: src/constants/folioToInnReachLocations.js (hunk 2 above)
Comment (Dmytro-Melnyshyn) [reply src/constants/folioToInnReachLocations.js]: Thanks, fixed

File: src/constants/sections.js (hunk 3 above)
Comment (Dmytro-Melnyshyn) [reply src/constants/sections.js]: Thanks, fixed

File: src/settings/routes/FolioToInnReachLocations/FolioToInnReachLocationsCreateEditRoute.js [hunk 4]
```diff
@@ -0,0 +1,572 @@
...
+  const servers = centralServers[0]?.centralServers || [];
+  const innReachLocations = innReachLoc[0]?.locations || [];
+  const folioLocations = locations[0]?.locations || [];
+  const folioLibraries = libraries[0]?.loclibs || [];
+
+  const { formatMessage } = useIntl();
+  const showCallout = useCallout();
+  const unblockRef = useRef();
+
+  // central server states
```
[the remark this answers is not in the document]
Comment (Dmytro-Melnyshyn) [reply src/settings/routes/FolioToInnReachLocations/FolioToInnReachLocationsCreateEditRoute.js]: Thanks, fixed

**Review by Godlevskyi — APPROVED** [review]

**Pull request merged**